platforms/Linux/shared/RPM/createrepo/createrepo_rpm.sh

```diff
@@ -0,0 +1,40 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+#!/usr/bin/env bash
+
+set -ex
+
+INDIR=/input
```

Comment:
shell variables are `supposed_to_be_lower_camel_case` usually. Only do all caps if they either have special meaning to the shell (like `PATH`) or if you accept them from the outside.